Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -384,51 +390,63 @@ public bool OpenSolution(string solutionFolder)
                     Reporter.ToLog(eLogLevel.WARN, "Loading Solution- Error: Current Ginger version can't load the Solution because it includes items from higher Ginger version");
                     return false;
                 }
-
+                
                 Reporter.ToLog(eLogLevel.DEBUG, "Loading Solution- Loading Solution xml into object");
                 Solution solution = Solution.LoadSolution(solutionFile);
+                Console.WriteLine("Solution Laaded Successfully");
                 if (solution == null)
                 {
                     Reporter.ToUser(eUserMsgKey.SolutionLoadError, "Failed to load the Solution file");
                     Reporter.ToLog(eLogLevel.WARN, "Loading Solution- Error: Failed to load the Solution file");
+                    Console.WriteLine("Solution is null");
                     return false;
                 }
 
                 Reporter.ToLog(eLogLevel.DEBUG, "Loading Solution- Creating Items Repository");
                 SolutionRepository = GingerSolutionRepository.CreateGingerSolutionRepository();
                 SolutionRepository.Open(solutionFolder);
 
+                Console.WriteLine("Solution loaded in SR");
+
                 Reporter.ToLog(eLogLevel.DEBUG, "Loading Solution- Loading needed Plugins");
                 mPluginsManager = new PluginsManager();
                 mPluginsManager.SolutionChanged(SolutionRepository);
 
+                Console.WriteLine("Solution Changed");
+
                 Reporter.ToLog(eLogLevel.DEBUG, "Loading Solution- Doing Source Control Configurations");
                 HandleSolutionLoadSourceControl(solution);
 
+                Console.WriteLine(" handle Source Control");
+
                 Reporter.ToLog(eLogLevel.DEBUG, "Loading Solution- Updating Application Functionalities to Work with Loaded Solution");
                 ValueExpression.SolutionFolder = solutionFolder;
                 BusinessFlow.SolutionVariables = solution.Variables; 
                 solution.SetReportsConfigurations();
+                Console.WriteLine("loaded Report Config");
                 Solution = solution;                
                 UserProfile.LoadRecentAppAgentMapping();
-                
+                Console.WriteLine("loaded Agent Mapping");
+
                 if (!RunningInExecutionMode)
                 {
                     AppSolutionRecover.DoSolutionAutoSaveAndRecover();   
                 }
 
+                Console.WriteLine("Autosave");
                 //Solution items upgrade
                 SolutionUpgrade.CheckSolutionItemsUpgrade(solutionFolder, solution.Name, solutionFiles.ToList());
-                
+
                 // No need to add solution to recent if running from CLI
                 if (!RunningInExecutionMode)  
                 {
                     UserProfile.AddSolutionToRecent(solution);
                 }
-
+                Console.WriteLine("solution added to recent");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)